### ISSUE TITLE
PolicyLengthExceededException

### ISSUE THREAD

**Issue by bjldb** [amplify-education/serverless-log-forwarding#25]
Hi,

I am using serverless-log-forwarding plugin for our project (1 serverless.yml, multiple functions as part of a step function which accesses other AWS services such as DynamoDB, S3, and Kinesis) and we have been encountering this error consistently when deploying to production. Any idea what our team might have missed?

```
  Serverless Error ---------------------------------------

  An error occurred: LogForwardingLambdaPermission - The final policy size (20711) is bigger than the limit (20480). (Service: AWSLambda; Status Code: 400; Error Code: PolicyLengthExceededException; Request ID: 80703966-284a-11e9-b913-196df554a7ee).
```

This could be related to this [fix](https://github.com/serverless/serverless/pull/5531), but it is only available in **serverless 1.36.3**. I am not sure, though, if the plugin upgrading the serverless version would resolve the issue. 

Cheers!

**Comment by bjldb** [comment]
Looks like this PR is aimed to resolve this same concern: https://github.com/amplify-education/serverless-log-forwarding/pull/22

**Comment by jconstance-amplify** [comment]
This should be resolved via #27. Please reopen if that doesn't seem to have fixed your problem.
